src/database/queries/userChangemakerPermissions/deleteOne.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM user_changemaker_permissions
```

Comment:
I believe this is the first time we're deleting anything from the database! Which brings to mind the question of audit logs. I can talk myself into thinking that's a separate, deferrable concern, but I want that to be a conscious decision.

The alternative, I think, would be for the database to have `not_before` and `not_after` columns, and "deleting" a permission would update the `not_after` from `NULL` to `now()`. That would give us history which we could surface someday, without building a whole separate audit log layer.

What do you think?

Reply:
Would it be enough to just have `not_after` at this point?  I'm not sure we have a use case for `not_before` yet.

I'm open to it, though obviously it results in more complex queries / the possibility that we'll fail to add an appropriate WHERE clause some day!

Reply:
Sure, until we have that use case it'd just be identical to `created_at` anyway, and when that time comes the migration could copy `created_at` into the new column.

Reply:
This sounds good to me. To be clear, the suggestion is to keep the HTTP delete endpoint as an HTTP delete, however, avoid scrubbing the database and use the above technique?

Reply:
@bickelj yes that's correct

Reply:
I'm going to move delete into a separate PR, so marking this as resolved for now.